test/e2e/scheduler_predicates.go

```diff
@@ -718,9 +717,8 @@ var _ = framework.KubeDescribe("SchedulerPredicates [Serial]", func() {
 
 		By("Trying to launch a pod that with NodeAffinity setting as embedded JSON string in the annotation value.")
 		labelPodName := "with-labels"
-		nodeSelectionRoot := filepath.Join(framework.TestContext.RepoRoot, "test/e2e/testing-manifests/node-selection")
-		testPodPath := filepath.Join(nodeSelectionRoot, "pod-with-node-affinity.yaml")
-		framework.RunKubectlOrDie("create", "-f", testPodPath, fmt.Sprintf("--namespace=%v", ns))
+		affinityPod := string(framework.ReadOrDie("test/e2e/node-selection/pod-with-node-affinity.yaml")[:])
```

Comment:
I'm guessing since this merged you'll want to change this back to `test-manifests`? https://github.com/kubernetes/kubernetes/pull/25564